src/transaction/log_recovery.c

```diff
@@ -5084,7 +5088,91 @@ log_recovery_undo (THREAD_ENTRY * thread_p)
 	    }
 	}
 
-      assert (log_Gl.m_active_start_position != 0);
+      _er_log_debug (ARG_FILE_LINE, "HA recovery: Finished recovery of min active stream position"
+		     "m_active_start_position=%llu, m_ack_stream_position=%llu",
+		     (std::uint64_t) log_Gl.m_repl_rv.m_active_start_position,
+		     (std::uint64_t) log_Gl.hdr.m_ack_stream_position);
+    }
+
+  if (!LSA_ISNULL (&log_Gl.m_min_active_lsa))
```

Comment:
Is it possible to merge this if block to the very above?

Reply:
removed the block above, and handled things in logtb_rv_assign_mvccid_for_undo_recovery